util/state_division_to_dcid.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Geo utility that maps the names of reporting areas in NNDSS to Data Commons
+place nodes.
+"""
+
+_PLACE_MAP = {
+    "U.S. Residents": "country/USA",
```

Comment:
may be easier to read/search if entries in this map are sorted alphabetically, either by key or value.

Reply:
I have sorted the dictionary by the key